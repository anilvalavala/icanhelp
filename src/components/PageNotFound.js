import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () => {
    return(
        <div>
            <h1>Requested page was not found</h1>
            <Link to="/">HOME</Link>
        </div>
    );
}

export default PageNotFound;