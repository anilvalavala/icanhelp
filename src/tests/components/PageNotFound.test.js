import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../components/PageNotFound';

test('Test PageNotFound component', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});