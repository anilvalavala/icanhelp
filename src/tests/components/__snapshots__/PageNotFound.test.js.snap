// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PageNotFound component 1`] = `
<div>
  <h1>
    Requested page was not found
  </h1>
  <Link
    replace={false}
    to="/"
  >
    HOME
  </Link>
</div>
`;
